pkg/entities/webhook.go

```diff
@@ -335,6 +334,20 @@ func (e *Entity) replyGmGn(streak *model.DiscordUserGMStreak, channelID, discord
 		return err
 	}
 
+	if streak.StreakCount == 1 {
```

Comment:
create a common var `embed` and reuse it for every condition since the only difference is `description`
```
&discordgo.MessageEmbed{
			Title:       "GM / GN",discordID),
			Color:       3447003,
			Footer: &discordgo.MessageEmbedFooter{
				Text:    authorUsername,
				IconURL: authorAvatar,
			},
			Timestamp: time.Now().Format("2006-01-02 15:04:05"),
		}
		```